ratis-client/src/main/java/org/apache/ratis/client/RaftClient.java

```diff
@@ -57,6 +58,9 @@
   /** @return the {@link MessageStreamApi}. */
   MessageStreamApi getMessageStreamApi();
 
+  /** @return the {@link BlockingApi}. */
+  BlockingApi getBlockingApi();
```

Comment:
Will `blocking()` better?

Reply:
blocking() may not be a good name.  How about io()?  It is similar to the java.io package.